integration-tests/07_course.ts

```diff
@@ -390,3 +393,44 @@ void test('Add / Remove another instructor', async () => {
         subcourseDeleteInstructor(subcourseId: ${subcourseId} studentId: ${instructor2.student.id})
     }`);
 });
+
+void test('Find shared course', async () => {
+    const { courseId, client } = await subcourseOne;
+
+    await client.request(`
+        mutation MarkCourseShared {
+            courseMarkShared(courseId: ${courseId}, shared: true){id}
+        }
+    `);
+
+    const { templateCourses } = await adminClient.request(`
+        query FindSharedCourses {
+            templateCourses(search: "", take: 1) {
+              id
+            }
+        }
+    `);
```

Comment:
There is currently no test that if an instructor shares a course, that another instructor can find it

Reply:
This test case is also added (line 429). This assert also tests if the searchh by course name works.